interfaces/builtin/core_support.go

```diff
@@ -50,6 +50,10 @@ const coreSupportConnectedPlugAppArmor = `
 /{,usr/}{,s}bin/sysctl                 ixr,
 @{PROC}/sys/{,**}                      r,
 @{PROC}/sys/**                         w,
+
+# Allow modifying logind configuration from the core snap
+/etc/systemd/logind.conf.d/{,*}         r,
+/etc/systemd/logind.conf.d/{,core.conf} w,
```

Comment:
IMHO this should go as just

    /etc/systemd/logind.conf.d/{,**} rw,

as this is the core snap which is already ultimatively trusted. That would make us safe from implementation details in hook and not bind us to single file in there. Nothing else can write in here yet and shouldn't.